### ISSUE TITLE
Crash with `/records?_since={}` 

### ISSUE THREAD

**Issue by leplatrem** [Kinto/kinto#2495]
https://sentry.prod.mozaws.net/operations/kinto-prod-1/issues/8328977/

```
ProgrammingError: can't adapt type 'dict'
  File "sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)

ProgrammingError: (psycopg2.ProgrammingError) can't adapt type 'dict'
[SQL: 
            SELECT id, as_epoch(last_modified) AS last_modified, data
            FROM objects
            WHERE parent_id = %(parent_id)s
            AND resource_name = %(resource_name)s
            
            AND as_epoch(last_modified) > %(filters_value_0)s
            
            ORDER BY last_modified DESC
            LIMIT %(pagination_limit)s;
        ]
[parameters: {'parent_id': '/buckets/main/collections/normandy-recipes-capabilities', 'resource_name': 'record', 'filters_value_0': {}, 'pagination_limit': 10001}]
(Background on this error at: http://sqlalche.me/e/f405)
(6 additional frame(s) were not displayed)
...
  File "sqlalchemy/engine/base.py", line 1287, in _execute_context
    self._handle_dbapi_exception(
  File "sqlalchemy/engine/base.py", line 1481, in _handle_dbapi_exception
    util.raise_(
  File "sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "sqlalchemy/engine/base.py", line 1247, in _execute_context
    self.dialect.do_execute(
  File "sqlalchemy/engine/default.py", line 590, in do_execute
    cursor.execute(statement, parameters)
```

**Comment by shivasheeshyadav** [comment]
Hi @leplatrem 
I would like to work on it.

**Comment by leplatrem** [comment]
Hi @shivasheeshyadav !
Did you manage to setup the dev environment?

**Comment by varsharmsh** [comment]
I would like to work on this issue.

**Comment by shivasheeshyadav** [comment]
Hi @leplatrem 
Working on it. 
@varsharmsh Could you please look into another issue.

**Comment by varsharmsh** [comment]
sure

**Comment by shivasheeshyadav** [comment]
Hi @leplatrem 
I've setup the dev env.
Could you please provide me credentials to https://sentry.prod.mozaws.net/operations/kinto-prod-1/issues/8328977/ so that I could get to know more details about the issue.

**Comment by leplatrem** [comment]
You should be able to reproduce with the PostgreSQL storage backend, by reaching out any plural endpoint with this filter: `/v1/buckets?_since={}`

Here is the Sentry:
![Screenshot_2020-06-09 BackendError ProgrammingError (psycopg2 ProgrammingError) can't adapt type 'dict'  SQL SELECT id, as_](https://user-images.githubusercontent.com/546692/84129629-7e322380-aa42-11ea-87ee-562bf29a2b62.png)

**Comment by slav0nic** [comment]
Maybe it will be more correct working with SQL query as text()  rather than python string with formating?
https://docs.sqlalchemy.org/en/13/core/tutorial.html#using-textual-sql

**Comment by sihan-tawsik** [comment]
Hello,
I want to work on this. Could anyone give me a heads up, please?

**Comment by shivasheeshyadav** [comment]
Hi @sihan-tawsik 
This issue is already addressed in #2525 and now being reviewed.
You could take up any other open issues.

**Comment by sihan-tawsik** [comment]
Okay. Thank you @shivasheeshyadav
